presto-docs/src/main/sphinx/sql/create-view.rst

```diff
@@ -20,6 +22,22 @@ referenced by another query.
 The optional ``OR REPLACE`` clause causes the view to be replaced if it
 already exists rather than raising an error.
 
+Security
+--------
+
+In the default ``DEFINER`` security mode, tables referenced in the view
+are accessed using the permissions of the view owner (the *creator* or
+*definer* of the view) rather than the user executing the query. This
+allows providing restricted access to the underlying tables, for which
+the query user may not be allowed to access directly. Note that the
+``current_user`` function will return the query user, not the view owner,
```

Comment:
I'm not sure it's all that clear what "the query user" means. You may want to take a look at how the spec refers to it.

Reply:
Will change to "the user executing the query" and move this note to a separate paragraph, also explaining that this applies to either mode